README.md

```diff
@@ -21,6 +21,70 @@ Note: To be able to take advantage of alternate `define` method name, you will a
 build tooling generates using the alternate.  An example of this is done in the [emberjs-build](https://github.com/emberjs/emberjs-build)
 project in the [babel-enifed-module-formatter plugin](https://github.com/emberjs/emberjs-build/blob/v0.4.2/lib/utils/babel-enifed-module-formatter.js).
 
+## Public API
+
+`loader.js` provides the following as named exports from a `loader` module:
+
+```ts
+interface LoaderModule {
+  /*
+    Return the list of module id's that are present in the registry
+  */
+  getIds(): string[];
```

Comment:
May just be me, but for some reason, I do not think `id` when I think of the identifier for a module. Usually `path`  or `name` instead. When I first saw `getIds()` I had no idea what information it was actually returning.